pom.xml

```diff
@@ -3243,6 +3264,31 @@
           </dependency>
         </dependencies>
       </dependencyManagement>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>net.alchim31.maven</groupId>
+              <artifactId>scala-maven-plugin</artifactId>
+              <configuration>
+                <args combine.children="append">
+                  <arg>-Wconf:cat=deprecation:wv,any:e</arg>
+                  <arg>-Wconf:cat=scaladoc:wv</arg>
+                  <arg>-Wconf:cat=unchecked:wv</arg>
+                  <arg>-Wconf:cat=feature:wv</arg>
+                  // TODO(SPARK-33499): Enable this option when Scala 2.12 is no longer supported.
```

Comment:
I think we should use `<!-- -->` XML syntax for comments.

Reply:
Thx~  Address e4b4e0f fix this